samples/Directory.Build.props

```diff
@@ -2,4 +2,6 @@
   <PropertyGroup>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
```

Comment:
Do we want samples to inherit root? They are standalone examples and we probably don't want root settings such as target frameworks impacting them.

Reply:
I've done it for consistency. Happy to pull it out, if you like.

Reply:
Yes please pull it out.